docs/source/tutorial_optimization.md

```diff
@@ -584,215 +479,146 @@ reference_mu = opt_fom_result.x
 
 ```{code-cell}
+from pymor.algorithms.tr import coercive_rb_trust_region
+from pymor.parameters.functionals import MinThetaParameterFunctional
+
+coercivity_estimator = MinThetaParameterFunctional(fom.operator.coefficients, mu_bar)
 pdeopt_reductor = CoerciveRBReductor(
     fom, product=fom.energy_product, coercivity_estimator=coercivity_estimator)
-opt_rom = pdeopt_reductor.reduce()
-```
-
-```{code-cell}
-def enrich_adaptively_and_compute_objective_function(mu, data, opt_dict):
-    opt_rom = opt_dict['opt_rom']
-    primal_estimate = opt_rom.estimate_error(opt_rom.parameters.parse(mu))
-    if primal_estimate > 1e-2:
-        print('Enriching the space because primal estimate is {} ...'.format(primal_estimate))
-        U = fom.solve(mu)
-        try:
-            pdeopt_reductor.extend_basis(U)
-            data['enrichments'] += 1
-            opt_rom = pdeopt_reductor.reduce()
-        except:
-            print('... Extension failed')
-    else:
-        print('Do NOT enrich the space because primal estimate is {} ...'.format(primal_estimate))
-    opt_rom = pdeopt_reductor.reduce()
-    QoI = opt_rom.output(mu)[0, 0]
-    return QoI, data, opt_rom
-```
-
-```{code-cell}
-opt_along_path_adaptively_minimization_data = prepare_data(enrichments=True)
 
-opt_dict = {'opt_rom': opt_rom}
-opt_along_path_adaptively_result = optimize(enrich_adaptively_and_compute_objective_function,
-                                            opt_along_path_adaptively_minimization_data, ranges,
-                                            gradient=partial(compute_gradient_with_opt_rom, opt_dict),
-                                            adaptive_enrichment=True, opt_dict=opt_dict)
-```
+tic = perf_counter()
+tr_mu, tr_minimization_data = coercive_rb_trust_region(pdeopt_reductor, parameter_space=parameter_space,
+                                                       initial_guess=np.array(initial_guess))
+toc = perf_counter()
 
-```{code-cell}
-report(opt_along_path_adaptively_result, opt_along_path_adaptively_minimization_data, reference_mu)
+tr_minimization_data['time'] = toc - tic
+tr_output = fom.output(tr_mu)[0, 0]
 ```
 
-Now, we actually only needed {math}`4` enrichments and ended up with an
-approximation error of about `1e-07` while getting the highest speed up
-amongst all methods that we have seen above. Note, however, that this is
-still dependent on the tolerance `atol=1e-2` that we chose without
-knowing that this tolerance suffices to reach the actual minimum.
-An easy way around this would be to do one optimization step with the FOM
-after converging. If this changes anything, the ROM tolerance `atol`
-was too large. To conclude, we once again
-compare all methods that we have discussed in this notebook.
+Now, we only need {math}`4` enrichments and end up with an approximation
+error of about `1e-06` which is comparable to the result obtained from the
+previous methods. Unfortunately, the trust-region method takes longer because of
```

Comment:
In the rendered documentation, the TR algorithm is the fastest. Otherwise, this should be phrased more positively.